engines/qemu/network/pool.go

```diff
@@ -362,8 +368,9 @@ func destroyNetwork(n *entry) error {
 		{"ip", "addr", "del", n.ipPrefix + ".1", "dev", n.tapDevice},
 		// Delete tap device
 		{"ip", "tuntap", "del", "dev", n.tapDevice, "mode", "tap"},
-	})
+	}, true)
 	if err != nil {
+		debug("Failed to destory tap device: %s, error: %s", n.tapDevice, err)
```

Comment:
Yeah, I'm not happy with the setup either... In particular I'm having some reliability issues...

So I might have to redo the network setup... Hence, I'm not going to go ahead with a huge documentation story on yet either,...